Sources/Convenience/Optional.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Optional.swift
+//  CombineExt
+//
+//  Created by Jasdev Singh on 11/05/2020.
+//  Copyright © 2020 Combine Community. All rights reserved.
+//
+
+import Combine
```

Comment:
All new code should be wrapped in a 

```swift
#if canImport(Combine)

#endif
```

I might need to document this but it's something we have to do to support installation on older versions